contracts/cw721-base/src/upgrades/v0_16.rs

```diff
@@ -0,0 +1,22 @@
+use cosmwasm_std::{CustomMsg, DepsMut, Response};
+use cw721_base_016 as v16;
+use serde::{de::DeserializeOwned, Serialize};
+
+use crate::ContractError;
+
+pub fn migrate<T, C, E, Q>(deps: DepsMut) -> Result<Response<C>, ContractError>
+where
+    T: Serialize + DeserializeOwned + Clone,
+    Q: CustomMsg,
+    E: CustomMsg,
+{
+    let tract16 = v16::Cw721Contract::<T, C, E, Q>::default();
+    let minter = tract16.minter.load(deps.storage)?;
+    tract16.minter.remove(deps.storage);
+    cw_ownable::initialize_owner(deps.storage, deps.api, Some(minter.as_str()))?;
+    let ownership = cw_ownable::get_ownership(deps.storage)?;
```

Comment:
We can make `initialize_owner` return the ownership info, so that you don't need to `get_ownership` here. Thoughts? I can make a new release of cw-ownable if you want to do it in this PR.

```rust
let ownership = cw_ownable::initialize_owner(deps.storage, deps.api, Some(minter.as_str()))?;
```

Reply:
That would be sick!

Reply:
@larry0x https://github.com/steak-enjoyers/cw-plus-plus/pull/12 :)

Reply:
@0xekez Just released cw-ownable v0.5.0 with this change